shared/src/test/scala/mouse/StringSyntaxTests.scala

```diff
@@ -29,7 +29,9 @@ import org.scalacheck.{Arbitrary, Gen}
 class StringSyntaxTests extends MouseSuite {
   test("StringOps.parseInt") {
     assertEquals("123".parseInt, 123.asRight[NumberFormatException])
-    assertEquals("blah".parseInt.leftMap(_.getMessage), "For input string: \"blah\"".asLeft)
+    assertEquals("blah".parseInt.leftMap(_.getMessage.replace("string ", "string: ")),
+                 "For input string: \"blah\"".asLeft
+    )
```

Comment:
Oh, is this a bug where behavior doesn't match? Can you make an issue so we fix this in Scala Native core 🙏 thanks!

Reply:
The only difference is in the lack of semicolons. That's so minor thing, but if you say so, let's try to fix it :)